archivemount: add page (#9354)
tldr-pages/tldr: pages/linux/archivemount.md

# archivemount

> > Mount an archive for access as a file system.

Reviewer: ```suggestion
> Mount an archive for access as a filesystem.
```

> > Mounts an archive for access as a file system.

Reviewer 2: ```suggestion
> Mount an archive for access as a file system.
```

> More information: <https://manned.org/archivemount>.

> - Mount an archive to mountpoint:

Reviewer 2: ```suggestion
- Mount an archive to a mountpoint:
```
I think this would be better

> `archivemount {{archive}} {{mountpoint}}`

Reviewer 2: ```suggestion
`archivemount {{path/to/archive}} {{mount_point}}`
```
I think this would be better.

> `archivemount {{path/to/archive}} {{mount_point}}`

Reviewer: I think this is more clear
```suggestion
`archivemount {{path/to/archive}} {{path/to/mount_point}}`
```
